doc/cephfs/disaster-recovery.rst

```diff
@@ -160,3 +160,63 @@ the range 0-(N_workers - 1), like so:
 
 It is important to ensure that all workers have completed the
 scan_extents phase before any workers enter the scan_inodes phase.
+
+Using an alternate metadata pool for recovery
+---------------------------------------------
+
+.. warning::
+
+   There has not been extensive testing of this procedure. It should be
+   undertaken with great care.
+
+If an existing filesystem is damaged and inoperative, it is possible to create
+a fresh metadata pool and attempt to reconstruct the filesystem metadata
+into this new pool, leaving the old metadata in place. This could be used to
+make a safer attempt at recovery since the existing metadata pool would not be
+overwritten.
+
+.. caution::
+
+   During this process, multiple metadata pools will contain data referring to
+   the same data pool. Extreme caution must be exercised to avoid changing the
+   data pool contents while this is the case. Once recovery is complete, the
+   damaged metadata pool should be deleted.
+
+To begin this process, first create the fresh metadata pool and initialize
+it with empty file system data structures:
+
+::
+
+    ceph fs flag set enable_multiple true --yes-i-really-mean-it
+    rados mkpool recovery
```

Comment:
Let's not encourage people to use "rados mkpool", "ceph osd pool create" is nicer and forces them to choose sane PG counts